azurerm/resource_arm_data_lake_store.go

```diff
@@ -0,0 +1,193 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/datalake/store/mgmt/2016-11-01/account"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmDataLakeStore() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDateLakeStoreCreate,
+		Read:   resourceArmDateLakeStoreRead,
+		Update: resourceArmDateLakeStoreUpdate,
+		Delete: resourceArmDateLakeStoreDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"location": locationSchema(),
+
+			"resource_group_name": resourceGroupNameSchema(),
+
+			"tier": {
+				Type:             schema.TypeString,
+				ForceNew:         false,
```

Comment:
we don't need to specify `false` for `ForceNew` so can we remove this line?

Reply:
yup not needed